python/llm/example/CPU/HF-Transformers-AutoModels/Model/distil-whisper/README.md

```diff
@@ -0,0 +1,86 @@
+```
+
+### 2. Run
+```
+python ./recognize.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --repo-id-or-data-path REPO_ID_OR_DATA_PATH --language LANGUAGE --chunk-length CHUNK_LENGTH --batch-size BATCH_SIZE
+```
+
+Arguments info:
+- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the Distil-Whisper model to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'distil-whisper/distil-large-v2'`.
+- `--repo-id-or-data-path REPO_ID_OR_DATA_PATH`: argument defining the huggingface repo id for the audio dataset to be downloaded, or the path to the huggingface dataset folder. It is default to be `'distil-whisper/librispeech_long'`.
+- `--language LANGUAGE`: argument defining language to be transcribed. It is default to be `english`.
+- `--chunk-length CHUNK_LENGTH`: argument defining the maximum number of chuncks of sampling_rate samples used to trim and pad longer or shorter audio sequences. For audio recordings less than 30 seconds, it can be set to 0 for better performance. It is default to be 15.
+- `--batch-size BATCH_SIZE`: argument defining the batch_size of pipeline inference, it usually equals of length of the audio divided by chunk-length. It is default to be 16.
+
+> **Note**: When loading the model in 4-bit, BigDL-LLM converts linear layers in the model into INT4 format. In theory, a *X*B model saved in 16-bit will requires approximately 2*X* GB of memory for loading, and ~0.5*X* GB memory for further inference.
+>
+> Please select the appropriate size of the Distil-Whisper model based on the capabilities of your machine.
+
+
+#### 2.1 Client
+On client Windows machine, it is recommended to run directly with full utilization of all cores:
+```powershell
+python ./recognize.py
+```
+
+#### 2.2 Server
+For optimal performance on server, it is recommended to set several environment variables (refer to [here](../README.md#best-known-configuration-on-linux) for more information), and run the example with all the physical cores of a single socket.
+
+E.g. on Linux,
+```bash
+# set BigDL-Nano env variables
+source bigdl-nano-init
+
+# e.g. for a server with 48 cores per socket
+export OMP_NUM_THREADS=48
+numactl -C 0-47 -m 0 python ./recognize.py
+```
+
+#### 2.3 Sample Output
+#### 2.3.1 Short-Form Transcription
+
+Model: [distil-whisper/distil-large-v2](https://huggingface.co/distil-whisper/distil-large-v2)
+
+Command line:
+```bash
+python ./recognize.py --repo-id-or-data-path 'hf-internal-testing/librispeech_asr_dummy' --chunk-length 0
+```
+Output:
+```log
+Inference time: xxxx s
+-------------------- Output --------------------
+[' Mr. Quilter is the apostle of the middle classes, and we are glad to welcome his gospel.']
+```
+
+#### 2.3.2 Long-Form Transcription
+
+Model: [distil-whisper/distil-large-v2](https://huggingface.co/distil-whisper/distil-large-v2)
+
+Command line:
+```bash
+python ./recognize.py
```

Comment:
data path?

Reply:
The default arguments are for the long sequence transcription. Do I need to show arguments explicitly here?